lib/Minz/Extension.php

```diff
@@ -86,12 +78,12 @@ public function uninstall() {
 	 * enabled by the extension manager).
 	 * @return void
 	 */
-	abstract public function init();
+	abstract public function init(): void;
```

Comment:
Trying with PHP 8.2 and an older version of *CustomCSS Extension* just to represent the case of existing libraries:

```
Fatal error: Declaration of CustomCSSExtension::init() must be compatible with Minz_Extension::init(): void in /var/www/FreshRSS/extensions/xExtension-CustomCSS/extension.php on line 4
```

For all the methods, which are not `final`, I do not believe we can change their signature